<ui version="4.0" >
 <class>GroupDialog</class>
 <widget class="QDialog" name="GroupDialog" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>711</width>
    <height>737</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Group Specimens</string>
  </property>
  <property name="modal" >
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3" >
   <item>
    <widget class="QCheckBox" name="m_autoMeanCheck" >
     <property name="text" >
      <string>Automatically Append A Group Mean</string>
     </property>
     <property name="checked" >
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2" >
     <item>
      <widget class="QGroupBox" name="groupBox" >
       <property name="title" >
        <string>Specimens</string>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout" >
        <item>
         <widget class="SpecimensView" name="m_specimensView" />
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2" >
       <item>
        <spacer name="verticalSpacer" >
         <property name="orientation" >
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0" >
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="m_addButton" >
         <property name="enabled" >
          <bool>false</bool>
         </property>
         <property name="text" >
          <string>>></string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="m_removeButton" >
         <property name="enabled" >
          <bool>false</bool>
         </property>
         <property name="text" >
          <string>&lt;&lt;</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer_2" >
         <property name="orientation" >
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0" >
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox_2" >
       <property name="title" >
        <string>Groups</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout" >
        <item>
         <widget class="GroupTreeWidget" name="m_groupsView" >
          <column>
           <property name="text" >
            <string>Name</string>
           </property>
          </column>
          <column>
           <property name="text" >
            <string>Size</string>
           </property>
          </column>
          <column>
           <property name="text" >
            <string>Plot Symbol</string>
           </property>
          </column>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_4" >
          <item>
           <widget class="QPushButton" name="m_newGrpBtn" >
            <property name="text" >
             <string>New Group</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="m_delGroupBtn" >
            <property name="enabled" >
             <bool>false</bool>
            </property>
            <property name="text" >
             <string>Delete Group</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3" >
     <item>
      <widget class="QLineEdit" name="m_exprLineEdit" />
     </item>
     <item>
      <widget class="QPushButton" name="m_exprButton" >
       <property name="text" >
        <string>Add to Selection</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox" >
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons" >
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>SpecimensView</class>
   <extends>QTableView</extends>
   <header>../../SelectNode/SpecimensModel.h</header>
  </customwidget>
  <customwidget>
   <class>GroupTreeWidget</class>
   <extends>QTreeWidget</extends>
   <header>../GroupTreeWidget.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>GroupDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel" >
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>GroupDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel" >
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
